src/lightning_fabric/plugins/precision/fsdp.py

```diff
@@ -16,7 +16,7 @@
 import torch
 from typing_extensions import Literal
 
-from lightning_fabric.plugins.precision.native_amp import MixedPrecision
```

Comment:
Could you also consider removing the prefix from fabrics native_amp module?

Reply:
I did this already. Can you point out what I missed exactly?